src/models/server-security-requirement.ts

```diff
@@ -0,0 +1,5 @@
+import { BaseModel } from "./base";
+
+export interface ServerSecurityRequirementInterface extends BaseModel {
+
+}
```

Comment:
`ServerSecurityRequirementInterface` is unnecessary because it should not be a model, but only type https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#securityRequirementObject